src/modules/iracing-schedule/ui/components/iracing-schedule-content.tsx

```
@@ -0,0 +1,6 @@
+import { TabsContent } from "@/components/ui/tabs";
+
+interface IRacingScheduleContentProps {}
```

Biome (2.3.13)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)